packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -76,25 +78,45 @@ storiesOf('Tabs', module)
     () => render({keyboardActivation: 'manual'})
   )
   .add(
-    'overflowMode: dropdown',
-    () => render({overflowMode: 'dropdown'})
+    'middle disabled',
+    () => render({disabledKeys: ['val2']})
   )
   .add(
-    'overflowMode: dropdown, density: compact',
-    () => render({overflowMode: 'dropdown', density: 'compact'})
+    'all disabled',
+    () => render({disabledKeys: ['val1', 'val2', 'val3', 'val4', 'val5']})
   )
   .add(
-    'overflowMode: dropdown, orientation: vertical',
-    () => render({overflowMode: 'dropdown', orientation: 'vertical'})
+    'resizeable',
+    () => (
+      <div style={{minWidth: '100px', width: '300px', padding: '10px', resize: 'horizontal', overflow: 'auto', backgroundColor: 'var(--spectrum-global-color-gray-50)'}}>
```

Comment:
Lets make the height like 400px or so, that way it's not flying off the top of the page when i shrink it

Reply:
good catch on the spacing for the dropdown button, lemme ask spectrum cuz the XD file actually doesn't show an example for icon tab picker. I'll do the 400px above as well

Reply:
It's still a little off, the picker is too close to the indicator line compared to a tab

Reply:
Are you referring specifically to the compact variant? That one is a bit finicky due to the focus ring and how I had to do that 3px padding, but I'll take another stab at trying to get it to work

Reply:
yeah, i'm happy to look at it with you some time tomorrow if that's helpful

Reply:
kk think I got it fixed, was also able to remove the 3px stuff since that was only required when I wasn't overriding Picker's focus ring